src/Microsoft.ML.Data/DataLoadSave/TransformerChain.cs

```diff
@@ -250,13 +250,15 @@ private static TransformerChain<ITransformer> Create(IHostEnvironment env, Model
         public static void SaveTo(this ITransformer transformer, IHostEnvironment env, Stream outputStream)
             => new TransformerChain<ITransformer>(transformer).SaveTo(env, outputStream);
 
-        public static TransformerChain<ITransformer> LoadFrom(IHostEnvironment env, Stream stream)
+        public static ITransformer LoadFrom(IHostEnvironment env, Stream stream)
```

Comment:
>ITransformer [](start = 22, length = 12)

Would it have been bad in the case where the source wasn't a chain (the `@"Model"` branch below, to just put it in a chain?

Reply:
I ended up doing the opposite of that - so the transform chain is never nested in the `Model` directory.

---
In reply to: [264740060](https://github.com/dotnet/machinelearning/pull/2858#discussion_r264740060) [](ancestors = 264740060)